cmd/feature_test.go

```
@@ -88,7 +101,16 @@ func (t *testHarness) iHaveAProjectDirectory() error {
 		return nil
 	}
 
-	return t.fs.MkdirAll(filepath.Join(t.path, "features"), 0755)
+	if err := t.fs.MkdirAll(filepath.Join(t.path, "features"), 0755); err != nil {
+		return nil
```

This should return the error